src/main/java/com/wojciechkocik/usage/service/CourseUsageServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.wojciechkocik.usage.service;
+
+import com.wojciechkocik.usage.dto.CourseUsageCreate;
+import com.wojciechkocik.usage.dto.DailyUsage;
+import com.wojciechkocik.usage.dto.DailyUsageResponse;
+import com.wojciechkocik.usage.entity.CourseUsage;
+import com.wojciechkocik.usage.repository.CourseUsageRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+/**
+ * @author Wojciech Kocik
+ * @since 22.03.2017
+ */
+@Slf4j
+@Service
+public class CourseUsageServiceImpl implements CourseUsageService {
```

Comment:
For me _Impl_ means _basic implementation_ of interface. Some of Spring Framework classes use this convention.
eg.
```java
public class AuthenticationTrustResolverImpl implements AuthenticationTrustResolver from org.springframework.security.authentication
public class PageImpl<T> extends Chunk<T> implements Page<T> from org.springframework.data.domain
public class JavaMailSenderImpl implements JavaMailSender from org.springframework.mail.javamail
```

I don't agree with that I don't need an interface. For example If I would to change the implementation then I have to change the name of class everywhere in my code. In big projects it may be confused.